### ISSUE TITLE
Latest .htaccess file causes redirect loop on install

### ISSUE THREAD

**Issue by mcheck** [evolution/wordpress#60]
Using evolution 1.0.15, the latest additions to the .htaccess file is causing a redirect loop on a fresh install at ```wp-admin/install.php```

I didn't check into where it is happening, but removing the added ```RewriteCond```s allows it to work as normal.

Happy to help track it down if you like.

**Comment by mcheck** [comment]
Actually I don't know what might be causing it. I removed the .htaccess rules, reset the permissions and I get the same thing. Since this is a staging system, I simply pushed out my local data to it: ```cap staging evolve:up``` and it all works. But I would like to know why the install no longer works.

**Comment by ericrasch** [comment]
@mcheck This issue is coming from the `<files install.php>`

A workaround that worked for Genesis is this which allows for the `install.php` to be executed on local environments: 

```
SetEnvIf Host ^local\. allow_wp_install
<files install.php>
  Order deny,allow
  Deny from all
  Allow from env=allow_wp_install
</files>
```

**Comment by mcheck** [comment]
Thanks, odd that this worked fine on staging for me.
What is the general workflow in setting up a new site - is the wb db already set from a deploy?

**Comment by EvanK** [comment]
That's right, as of 1.0.15, we are [denying access to the install.php](https://github.com/evolution/wordpress/blob/9244154ac6ab2e29b5c09378607793de9e2f6b30/lib/ansible/roles/apache/templates/virtualhost#L88) for non-local environments. 

The intention being that an installation should be handled on the local stage, and then a deploy and db:up would get the db initialized for staging or production.

We are doing this as part of the virtualhost template, so it would only occur on a machine that was _provisioned_ with .15 or later

**Comment by EvanK** [comment]
Closing out, as this should be addressed by the workflow documentation in #70
